clients/unshaded/src/main/java/tachyon/client/file/FileOutStream.java

```diff
@@ -70,17 +71,17 @@
   protected BufferedBlockOutStream mCurrentBlockOutStream;
   protected List<BufferedBlockOutStream> mPreviousBlockOutStreams;
 
-  protected final long mFileId;
+  protected final TachyonURI mPath;
```

Comment:
Should we call this mUri instead? It's odd to have `mPath.getPath()` and `mPath.toString()` be different